includes/functions/functions_products.php

```diff
@@ -127,3 +126,169 @@ function zen_enable_disabled_upcoming() {
         zen_set_disabled_upcoming_status($disabled_upcoming_fields['products_id'], 1);
     }
 }
+
+
+/**
+ * Return a product's category (master_categories_id)
+ * @param int $products_id
+ * @return int|string
+ */
+function zen_get_products_category_id($products_id) {
```

Comment:
This line isn't factored as others, recommend:
````diff
- function zen_get_products_category_id($products_id) {
+ function zen_get_products_category_id($products_id)
+{
````

Reply:
Thanks. File updated.